src/array.jl

```diff
@@ -20,6 +20,9 @@ end
 CuVector{T} = CuArray{T,1}
 CuMatrix{T} = CuArray{T,2}
 CuVecOrMat{T} = Union{CuVector{T},CuMatrix{T}}
+CuMatOrAdj{T} = Union{CuMatrix, LinearAlgebra.Adjoint{T, CuMatrix{T}}, LinearAlgebra.Transpose{T, CuMatrix{T}}}
+CuOrAdj{T} = Union{CuVecOrMat, LinearAlgebra.Adjoint{T, CuVecOrMat{T}}, LinearAlgebra.Transpose{T, CuVecOrMat{T}}}
```

Comment:
Are you happy with this?

Reply:
Better add these type aliases to the linalg file

Reply:
Moved.